gruntfile.js

```diff
@@ -237,7 +237,27 @@ module.exports = function(grunt) {
           src: ['**'],
           dest: 'tmp/samples'
         }]
-      }
+      },
+      chrome_agent: {
+        files: [{
+          expand: true,
+          cwd: 'chrome_agent',
+          src: ['**'],
```

Comment:
I think that everything should be copied here.  Since this is an extension, we don't know what kinds of source might be here for sure (assets, etc).